terraform/modules/emrfs-lambda/policy-munge-lambda-files/tests/lambda_handler_tests.py

```diff
@@ -93,17 +93,18 @@ def test_get_env_vars(self, mock_get_env):
 
     @patch('lambda_handler.execute_statement')
     def test_get_user_userstatus_policy_dict(self, mock_execute_statement):
-        mock_execute_statement.side_effect = [mocked_db_response, {'numberOfRecordsUpdated': 0,'records': []}]
+        mock_execute_statement.side_effect = [mocked_db_response, {'numberOfRecordsUpdated': 0, 'records': []}]
         result1 = lambda_handler.get_user_userstatus_policy_dict(variables)
 
+        print(result1)
```

Comment:
Do we need these still? If so, should be one with a nice log statement describing it.